rllib/connectors/tests/test_agent.py

```diff
@@ -161,14 +162,11 @@ def test_state_buffer_connector(self):
 class TestViewRequirementConnector(unittest.TestCase):
```

Comment:
Several pieces of our code, including this test case, mention "ViewRequirementConnector", but we only offer "ViewRequirementAgentConnector"

Reply:
done